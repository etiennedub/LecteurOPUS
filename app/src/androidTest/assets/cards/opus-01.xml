<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<Opus>
  <ID>0000000000000004007100000000000093369BCA02500A440025433F009000</ID>
  <Expiration>04789200091C3C2A83A33260A0E2C17C1B1500000000000000000000009000</Expiration>
  <Transit>
    <List index="0">78EE7F80000CF000284C4809E847BC773FCA00000000000000000000009000</List>
    <List index="1">78EE2400000CF000282E4809E847BC77120A00000000000000000000009000</List>
    <List index="2">78E9E680000CF00028311002F847BC74F34A00000000000000000000009000</List>
  </Transit>
  <Subscription>
    <List index="0">1E0A0049BC62F27FF48DE32400000000000020000000000000000000009000</List>
    <List index="1">1E0A0049BBECF0A7F48DE01644000000000020000000000000000000009000</List>
    <List index="2">1E04038F80000007F481C837EC000000000000000000000000000000009000</List>
  </Subscription>
  <Ticket>
    <List index="0">0000000000000000000000FFFFFF0000000000000000000000000000009000</List>
    <List index="1">0000000000000000000000FFFFFF0000000000000000000000000000009000</List>
    <List index="2">0000010000000000000000FFFFFF0000000000000000000000000000009000</List>
  </Ticket>
</Opus>
