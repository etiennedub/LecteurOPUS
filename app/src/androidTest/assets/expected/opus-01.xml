<?xml version='1.0' encoding='UTF-8' standalone='yes' ?><Card><Type>OPUS</Type><ID>2469829578</ID><ExpirationDate>30/09/2020</ExpirationDate><Trip><List index="0" Date="11/03/2018" Time="22:19" Transfert="false" OperatorId="20" BusId="317" OperatorName="RTC" BusName="METROBUS 807" /><List index="1" Date="11/03/2018" Time="19:16" Transfert="false" OperatorId="20" BusId="317" OperatorName="RTC" BusName="METROBUS 807" /><List index="2" Date="10/03/2018" Time="16:13" Transfert="false" OperatorId="20" BusId="95" OperatorName="RTC" BusName="METROBUS 800" /></Trip><Contract><List index="0" Subscription="true" Valid="true" NbTicket="0" OperatorId="20" OperatorName="RTC" ValidityDate="31/03/2018" /><List index="1" Subscription="true" Valid="false" NbTicket="0" OperatorId="20" ValidityDate="31/01/2018" /><List index="2" Subscription="false" Valid="true" NbTicket="1" OperatorId="8" OperatorName="STM" /></Contract></Card>

